src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -39,7 +40,7 @@ public MovieSessionController(MovieSessionService movieSessionService,
     }
 
     @PostMapping
-    public MovieSessionResponseDto add(@RequestBody MovieSessionRequestDto requestDto) {
+    public MovieSessionResponseDto add(@RequestBody @Valid MovieSessionRequestDto requestDto) {
```

Comment:
here I think you also need to add @Valid annotation, do you set restrictions in the DTO for these fields?